src/main/java/ui/windows/StatsController.java

```diff
@@ -23,32 +25,35 @@ public void setDisplay(StatsDisplay display) {
 
     /**
      * Requests statistics for a given user and updates display.
+     * @param player the PlayerInformation of the user
      * @param username the name of the user
      * @return false if no display is supplied, true otherwise
      */
-    public boolean tryRequestUser(String username) {
+    public boolean tryRequestUser(PlayerInformation player, String username) {
```

Comment:
the controller shouldnt receive player information, it should retrieve player information and then pass it to the display as a StatsDisplayData object

Reply:
Should StatsController be part of UI package or Controllers package. If it is part of controllers, this would be fine and the file's package should change, but if not then PlayerInformation should not be included in this file.

Reply:
its a bit ambiguous right now, but im thinking it (and all the other controllers in the windows folder) should be in the controller package